corehq/apps/style/templates/style/bootstrap3/base.html

```diff
@@ -508,6 +508,16 @@
         {% endcompress %}
         {% endif %}
 
+        {% if request.use_timeago %}
+        <script src="{% new_static 'jquery-timeago/jquery.timeago.js' %}"></script>
```

Comment:
the `if` statement can't be in the compress tag, so probably good to leave it here